lib/BalanceTransaction.php

```diff
@@ -17,7 +17,7 @@
  * @property \Stripe\StripeObject[] $fee_details
  * @property int $net
  * @property string $reporting_category
- * @property string|null $source
+ * @property string|\Stripe\StripeObject|null $source
```

Comment:
Right, expandable polymorphic fields are not very well handled at the moment and are typed as `StripeObject` instead of the actual list of possible types.

We have a workaround/hack to handle this in `@return` type annotations, but it wasn't easy to reuse in this case. I'll try and fix later.